main/Main.hs

```diff
@@ -230,6 +231,9 @@ main = do
          refDbStructure
          refIsWorkerOn)
 
+chomp :: ByteString -> ByteString
+chomp bs = fromMaybe bs (BS.stripSuffix "\n" bs)
```

Comment:
I was trying to think of a shorter way to write this but this seems to be best.

Also how about defining `chomp` in a where clause of `extractAndTransform` rather than as a top level function? The reason I say this is no other functions use this helper.

Reply:
> I was trying to think of a shorter way to write this but this seems to be best.

Actually, this short form was suggested by the linter. What I wrote was much longer, using a `case`.

> Also how about defining `chomp` in a where clause of `extractAndTransform` rather than as a top level function? The reason I say this is no other functions use this helper.

The way I see it, chomp should be moved to a separate file as a (potentially) reusable utility. I find that `extractAndTransform` is already hairy enough.

`chomp` is based on [the function of the same name in Perl](http://perldoc.perl.org/functions/chomp.html), which is a top-level utility.

Reply:
Yeah I'm familiar with it from perl. The haskell `Text` type has `strip` which will drop whitespace including newlines from before and after a string, but ByteString doesn't have it.

Trying to think of a good place for this function. Was thinking `Types.hs` for a second, but that's more for our internal types and their helper functions.  Could make a whole new file seeing that this is a general purpose helper, but that seems premature because no other functions in the codebase need chomp. That's why I was suggesting to make it private to the function who uses it.

Reply:
I am fine with either. I will make it private for now.